### ISSUE TITLE
Enabling Namespace returns error

### ISSUE THREAD

**Issue by cwilkers** [kubernetes-sigs/federation-v2#653]
**What happened**:

```
$ kubefed2 -v 2 enable namespace
I0314 15:49:51.276320   24230 enable.go:199] Found resource "namespaces.core/v1"
F0314 15:49:51.510322   24230 enable.go:109] error: Error initializing validation schema accessor: Error loading openapi schema: SchemaError(io.k8s.federation.multiclusterdns.v1alpha1.IngressDNSRecordList.metadata): invalid object doesn't have additional properties
```

**What you expected to happen**:

Creation of `federatednamespaces` custom resource definition.

**How to reproduce it (as minimally and precisely as possible)**:

Federation v 0.0.6 on an OpenShift 4.0 test cluster, follow directions in user guide to set up federation (shortcut `hack/intall-latest.yaml`). After installing, attempt above command to create Namespaces federated type.

**Anything else we need to know?**:

**Environment**:
- Kubernetes version (use `kubectl version`)

```
$ oc version
Client Version: version.Info{Major:"4", Minor:"0+", GitVersion:"v4.0.22", GitCommit:"219bbe2f0c", GitTreeState:"", BuildDate:"2019-03-10T22:23:11Z", GoVersion:"", Compiler:"", Platform:""}
Server Version: version.Info{Major:"1", Minor:"12+", GitVersion:"v1.12.4+0f8e04e", GitCommit:"0f8e04e", GitTreeState:"clean", BuildDate:"2019-03-04T15:33:19Z", GoVersion:"go1.10.8", Compiler:"gc", Platform:"linux/amd64"}
```
- Federation version
```
$ oc -n federation-test logs federation-controller-manager-5fb55c99bb-cxw82 
Federation v2 controller-manager version: version.Info{Version:"0.0.6", GitCommit:"f3bec03f890ba89079d819213eef4d003df7ab8a", GitTreeState:"clean", BuildDate:"2019-03-12T22:30:29Z", GoVersion:"go1.11", Compiler:"gc", Platform:"linux/amd64"}

$ kubefed2 version
kubefed2 version: version.Info{Version:"v0.0.6", GitCommit:"d56dfb3b83e8d40e584b5e66116de98328846fd2", GitTreeState:"clean", BuildDate:"2019-02-21T18:17:51Z", GoVersion:"go1.11.2", Compiler:"gc", Platform:"linux/amd64"}
```
- Install method (helm vs script)

Script first, then attempted with operator, both with same result.

- Install configuration (namespaced vs cluster-scoped)

namespaced

- Others

/kind bug

**Comment by xunpan** [comment]
@cwilkers do you try the latest code? I tried in my latest updated code, it seems has no such issue.

Another question: if you use script to install federation, namespace should be federated. So, could you help to show your installation step?

**Comment by marun** [comment]
This bug was fixed by #646, closing.
